pkgs/os-specific/linux/firmware/intel2200BGFirmware/default.nix

```diff
@@ -1,18 +1,28 @@
-{ lib, fetchzip }:
+{ stdenv
+, lib
+, fetchurl }:
 
 let version = "3.1"; in
```

Comment:
Either remove this line or use `inherit version;` with a non rec set below. Probably missed this when cleaning up?

Reply:
Oops, indeed I intended to remove that line